### DISCUSSION TITLE
I have a student mail but i cant get student pack

### DISCUSSION THREAD

**Discussion by SeneSatka** [community/community#138583] [GitHub Education] [answered]
### Hi everyone,

Hello, I have a student mail but i havent a student card. I want to get student pack but system is wanting student id. And system only accept camera. How I can student pack?

**Comment by poyrazavsever** [comment]
If you don't have anything to prove you are a student you can't get it. You need to get a document from the school that shows you are in school or it may not happen.

**Comment by SeneSatka** [comment]
But i have student mail

**Comment by poyrazavsever** [comment, marked as answer]
You need to get a document from the school. They can give you a student certificate if you want.

**Comment by SeneSatka** [comment]
Okey thanks

**Comment by Dogancan33** [comment]
I have everything to apply for the student package, but somehow, it asks for the location when I allow it. Error getting the location. I have been trying on different computers, browsers etc but it doesn't work

**Comment by fadel11-hub** [comment]
I have everything I need to register for the student package, but somehow, it asks for my location when I allow it, and it’s already time to get the location. When I try to submit it, I get a rejection instead. Why is that? ...
